src/cli/rustup_mode.rs

```diff
@@ -60,7 +60,9 @@ pub fn main() -> Result<()> {
         ("default", Some(m)) => default_(cfg, m)?,
         ("toolchain", Some(c)) => match c.subcommand() {
             ("install", Some(m)) => update(cfg, m)?,
-            ("list", Some(m)) => handle_epipe(toolchain_list(cfg, m))?,
+            ("list", Some(m)) => {
+                handle_epipe(common::list_toolchains(cfg, m.is_present("verbose")))?
+            }
```

Comment:
This change isn't really in keeping with the rest of this match, I'd rather leave the expansion of `m.is_present("verbose")` to within a local `toolchain_list()`